tests/test_wave_01.py

```diff
@@ -116,14 +116,17 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     updated_data = watch_movie(janes_data, MOVIE_TITLE_1)
 
     # Assert
-    assert len(updated_data["watchlist"]) is 0
-    assert len(updated_data["watched"]) is 1
+    assert len(updated_data["watchlist"]) == 0
+    assert len(updated_data["watched"]) == 1
+    
     
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
+    assert MOVIE_TITLE_1 == updated_data["watched"][0]['title']
```

Comment:
In this case where the code is checking individual keys of the dictionary, I recommend including all of the relevant keys.
```suggestion
    assert MOVIE_TITLE_1 == updated_data["watched"][0]['title']
    assert GENRE_1 == updated_data["watched"][0]['genre']
    assert RATING_1 == updated_data["watched"][0]['rating']
```